identity/src/main/java/com/stripe/android/identity/ui/ConsentScreen.kt

```diff
@@ -243,17 +237,35 @@ private fun SuccessUI(
 
         var acceptState by remember { mutableStateOf(LoadingButtonState.Idle) }
         var declineState by remember { mutableStateOf(LoadingButtonState.Idle) }
+        var scrolledToBottom by remember { mutableStateOf(false) }
 
-        LoadingButton(
-            modifier = Modifier
-                .padding(bottom = 10.dp)
-                .semantics { testTag = acceptButtonTag },
-            text = consentPage.acceptButtonText.uppercase(),
-            state = acceptState
-        ) {
-            acceptState = LoadingButtonState.Loading
-            declineState = LoadingButtonState.Disabled
-            onConsentAgreed(requireSelfie)
+        if (scrollState.value == scrollState.maxValue) {
+            scrolledToBottom = true
+        }
+
+        if (scrolledToBottom) {
```

Comment:
Can we not use the same button and just update its enabled state and text based on the `scrolledToBottom`?